### PR TITLE
Change complete regular expression for every request based on recent Rails Logs

### PR DESCRIPTION
**Pull request by lavika**

### CODE REVIEW

**Comment by sukeerthiadiga** [comment]
👍

File: avmlog.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var ntlm_regexp      *regexp.Regexp = regexp.MustCompile(" (\\(NTLM\\)|NTLM:) ")
 var debug_regexp     *regexp.Regexp = regexp.MustCompile(" DEBUG ")
 var error_regexp     *regexp.Regexp = regexp.MustCompile("( ERROR | Exception | undefined | NilClass )")
 
-var complete_regexp  *regexp.Regexp = regexp.MustCompile(" Completed ([0-9]+) [A-Za-z ]+ in ([0-9.]+)ms \\(Views: ([0-9.]+)ms \\| ActiveRecord: ([0-9.]+)ms\\)")
+var complete_regexp  *regexp.Regexp = regexp.MustCompile(" Completed ([0-9]+) [A-Za-z ]+ in ([0-9.]+)ms \\(Views: ([0-9.]+)ms \\| ActiveRecord: ([0-9.]+)ms \\| Allocations: [0-9.]+\\)")
```
Comment (smlsml) [inline avmlog.go:33]: Just curious, what does allocations indicate? I assume you all added that to the log lines for something. I'm happy to help compile a new version and release with this in there.

Comment (lavika) [reply avmlog.go:33]: Hi Steve,

With https://github.com/rails/rails/pull/33449, Rails started logging allocations which tells about how many objects were allocated during the event.

**Review by zeeamber — APPROVED** [review]
LGTM.

**Pull request merged**